src/tests/unit_tests/test_config.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from sherpa_ai.config import AgentConfig
+
+
+def test_parse_args():
```

Comment:
Try to name the tests to say exactly what they verify. In this case, I would do something like this:
```
test_parse_args -> test_all_parameters_parse_successfully
test_parse_args_partial -> test_no_gsite_parses_successfully
test_parse_args_noise -> test_invalid_input_raises_value_error
```